chief.py

```diff
@@ -72,12 +72,11 @@ def history(status):
         output = open(log_file, 'a')
 
         pub('BEGIN')
-        notify('%s is pushing %s - %s' % (who, app_name, webapp_ref))
-        yield '%s is pushing %s - %s\n' % (who, app_name, webapp_ref)
+        yield prefix_notify('%s is pushing %s - %s\n' % (who, app_name, webapp_ref))
 
         if getattr(settings, 'LOG_ROOT', None):
-            yield prefix_notify('%s/%s/logs/%s' % (settings.LOG_ROOT,
-                                             app_name, log_name)) + '\n'
+            yield prefix_notify('%s/%s/logs/%s\n' % (settings.LOG_ROOT,
+                                             app_name, log_name))
```

Comment:
NIT: the indentation on this line is odd. Normally the `app_name` and `settings` would line up. No biggie, but I'd bet flake8 would complain.